app.js

```diff
@@ -5,6 +5,20 @@ var app = express();
 var mongoose = require('mongoose');
 var bodyParser = require('body-parser');
 var methodOverride = require('method-override');
+var bunyan = require('bunyan');
+var log = bunyan.createLogger({
+                name: 'bsafe',
+                streams: [
+                    {
+                        level: 'info',
+                        path: __dirname + '/logs/info.log'
```

Comment:
Try to put the log files in some standard place like `/var/logs/`. Let's google around what's the standard location where other apps keep the logs.